dev-infrastructure/modules/acr-permissions.bicep

```diff
@@ -57,10 +52,33 @@ resource acrDeleteRole 'Microsoft.Authorization/roleAssignments@2022-04-01' = if
   }
 }
 
-resource acrContributorRole 'Microsoft.Authorization/roleAssignments@2022-04-01' = if (grantContributorAccess) {
-  name: guid(acrResourceGroupid, principalId, contributorRoleDefinitionId)
+resource tokenManagementRole 'Microsoft.Authorization/roleDefinitions@2022-04-01' = if (grantManageTokenAccess) {
+  name: guid(acrResourceGroupid, principalId, 'token-creation-role')
```

Comment:
what is the scope of custom roles? resourcegroup/subscription/tenant?
should we define our role on subscription level?

Reply:
I think you find your answer in line 58 in this file and in line 372 in `svc-cluster.bicep`